devel/gdb/Portfile

```
@@ -90,6 +90,14 @@ post-destroot {
     }
 }
 
+# These patches should be removed once they get merged into mainstream.
+# Fingers crossed they get merged in for gdb 11.3
+# https://sourceware.org/bugzilla/show_bug.cgi?id=24069
+if {${os.platform} eq "darwin" } {
```

Note that this can be written more succinctly:
```suggestion
platform darwin {
```

---

```
$ port lint --nitpick devel/gdb
--->  Verifying Portfile for gdb
Error: Failed to lint gdb: can't read "platform_arch": no such variable
Error: Follow https://guide.macports.org/#project.tickets if you believe there is a bug.
Error: Processing of port gdb failed
[Exit 1]
```

I'm afraid I'm not sure why this syntax isn't working, if you have any pointers I could look into, I'd be happy to change things to match your suggestion. Either way, thanks for the protip!

---

Seems like I had a umask problem. Now that that's fixed, I was able to use the syntax you suggested